app/src/org/commcare/android/framework/CommCareActivity.java

```diff
@@ -717,4 +723,11 @@ public Spannable localize(String key){
     public Spannable localize(String key, String[] args){
         return MarkupUtil.localizeStyleSpannable(this, key, args);
     }
+
+    @TargetApi(Build.VERSION_CODES.HONEYCOMB)
+    public void refreshBreadcrumbBar() {

```

Comment:
I would call this refreshActionBar, since the action bar really isn't used for breadcrumbs much anymore.